readme.md

```diff
@@ -145,6 +145,16 @@ You can pass an empty string to remove the suffix.
 
 For example, on macOS, the config file will be stored in the `~/Library/Preferences/foo-nodejs` directory, where `foo` is the `projectName`.
 
+#### watch
+
+type: `boolean`<br>
+Default: `false`
+
+Sets a file watcher.
+
+This is useful if you have more than one instance reading the config. If one instance changes a config, this allows it to get propagated to all other instances. If you only ever have one instance in your app you can keep this off.
```

Comment:
It's not really just about instances, as I cannot see why anyone would have multiple instances in the same process. This is about getting updates between different processes.